### ISSUE TITLE
[Support Request] Should the Build Guide be updated?

### ISSUE THREAD

**Issue by linglilongyi** [daeuniverse/dae#559]
### Checks

- [X] I have searched the existing issues
- [X] I have read the documentation
- [X] Is it your first time sumbitting an issue

### Support Request

Have trouble on building my own dae.

### Current Behavior

log:
```
go/pkg/mod/github.com/cilium/ebpf@v0.12.3/internal/sysenc/marshal.go:37:29: undefined: unsafe.StringData
note: module requires Go 1.20
```
while the [[https://github.com/daeuniverse/dae/blob/main/docs/en/user-guide/build-by-yourself.md|Build Guide Document]] mention `golang >= 1.18`

### Steps to Reproduce

make GOFLAG="-buildvcs=false" CC=clang

### Environment

- **Dae version (use `dae --version`)**: main
- **OS (e.g `cat /etc/os-release`)**:Debian 12
- **Kernel (e.g. `uname -a`)**:6.1.0-21-amd64
- **Others**:

### Anything else?

I would like to help improve the doc if the infomation is provided. Thank you all~

**Comment by mzz2017** [comment]
Indeed, dae needs go 1.22 from now

**Comment by mzz2017** [comment]
The go version needed can be found at https://github.com/daeuniverse/dae/blob/main/go.mod#L3

Welcome to contribute to the doc! Thank you.

**Comment by mzz2017** [comment]
Fixed in #561
